### PR TITLE
Feat/material ui basic home page

### PR DESCRIPTION
**Pull request by acenine**

### CODE REVIEW

File: client/src/components/Payment.jsx [hunk 1]
```diff
@@ -60,43 +83,136 @@ class Payment extends React.Component {
...
+  //             backgroundColor="#3D95CE"
+  //             hoverColor='#03A9F4'
+  //             labelStyle={style.button.lable}
+  //           />
+  //         </div>
+  //       </div>
+  //     </Paper>
+  //   );
+  // }
+
```
Comment (gingeranyhow) [inline client/src/components/Payment.jsx]: Remove commented out code, unless you want to keep this for some reason?

File: client/src/components/Payment.jsx [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+const style = {
+  form: {
+    display: 'block',
+  },
+  input: {
+    background: '#f8f8f8',
+  },
+  button: {
+    lable: {
+      color: '#fff',
```
Comment (gingeranyhow) [inline client/src/components/Payment.jsx]: change "lable" to "label" in all cases.

File: client/dist/styles.css [hunk 3]
```diff
@@ -92,3 +111,46 @@ button {
...
+.payment-item-container {
+  display: 'block';
+  justify-content: 'space-evenly';
+}
+.form-box {
+  padding-bottom: 20px;
+}
+.bottom {
+}
+.top {
```
Comment (gingeranyhow) [inline client/dist/styles.css:148]: empty class.

Comment (gingeranyhow) [reply client/dist/styles.css:148]: I think we might want to use more specific classnames than "top, left, right'. Thoughts?

File: client/src/components/Payment.jsx (hunk 1 above)
Comment (acenine) [reply client/src/components/Payment.jsx]: I am keeping it incase I need to change the styling. I'll remove it when the final decision is made on the payment component styling.

**Pull request merged**